src/libraries/System.Formats.Tar/src/Resources/Strings.resx

```diff
@@ -0,0 +1,252 @@
+  </data>
+  <data name="IO_SetLengthRequiresSeekingAndWriting" xml:space="preserve">
+    <value>SetLength requires a stream that supports seeking and writing.</value>
+  </data>
+  <data name="IO_SharingViolation_File" xml:space="preserve">
+    <value>The process cannot access the file '{0}' because it is being used by another process.</value>
+  </data>
+  <data name="IO_SharingViolation_NoFileName" xml:space="preserve">
+    <value>The process cannot access the file because it is being used by another process.</value>
+  </data>
+  <data name="IO_StreamDisposed" xml:space="preserve">
+    <value>Cannot access a disposed stream.</value>
+  </data>
+  <data name="IO_StreamNotEmpty" xml:space="preserve">
+    <value>The stream is not empty.</value>
+  </data>
+  <data name="PlatformNotSupported_SystemFormatsTar" xml:space="preserve">
+    <value>System.Formats.Tar is not supported on this platform.</value>
+  </data>
+  <data name="SetLengthRequiresSeekingAndWriting" xml:space="preserve">
+    <value>SetLength requires a stream that supports seeking and writing.</value>
+  </data>
+  <data name="TarDuplicateExtendedAttribute" xml:space="preserve">
+    <value>The entry '{0}' has a duplicate extended attribute.</value>
+  </data>
+  <data name="TarUnexpectedMetadataEntry" xml:space="preserve">
+    <value>A metadata entry of type '{0}' was unexpectedly found after a metadata entry of type '{1}'.</value>
+  </data>
+  <data name="TarEntriesInDifferentFormats" xml:space="preserve">
+    <value>The archive contains entries in different formats.</value>
+  </data>
+  <data name="TarEntryBlockOrCharacterExpected" xml:space="preserve">
+    <value>Cannot set the DevMajor or DevMinor fields on an entry that does not represent a block or character device.</value>
+  </data>
+  <data name="TarEntryHardLinkOrSymLinkExpected" xml:space="preserve">
+    <value>Cannot set the LinkName field on an entry that does not represent a hard link or a symbolic link.</value>
+  </data>
+  <data name="TarEntryModeOutOfRange" xml:space="preserve">
+    <value>The mode must be a base 10 number between 0 and 511 (777 in octal).</value>
+  </data>
+  <data name="TarEntryNotARegularFile" xml:space="preserve">
+    <value>Entry type is not a regular file, so it does not support setting the data stream.</value>
+  </data>
+  <data name="TarEntryTypeNotSupported" xml:space="preserve">
+    <value>Entry type '{0}' not supported in format '{1}'.</value>
+  </data>
+  <data name="TarEntryTypeNotSupportedForExtracting" xml:space="preserve">
+    <value>Entry type not supported for extraction: '{0}'</value>
+  </data>
+  <data name="TarExtractingResultsInOutside" xml:space="preserve">
+    <value>Extracting Tar entry would have resulted in a file outside the specified destination directory.</value>
+  </data>
+  <data name="TarGnuFormatExpected" xml:space="preserve">
+    <value>A GNU format was expected, but could not be reliably determined for entry '{0}'.</value>
```

Comment:
```suggestion
    <value>Entry '{0}' declared it is in a GNU format but did not have the expected version data.</value>
```
?

Reply:
There is a typo in the suggestion, but I changed to a very similar string.